### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by rozap**
* Remove all the console.logs
* Added bunyan
* Added log wrapper
* req and response have log methods, which eventually will log the request id
* not sure of a good way to test because node doesn't have a way to capture IO

### CODE REVIEW

File: lib/boot.js [hunk 1]
```diff
@@ -2,10 +2,6 @@ import service from './service';
 import config from './config';
 import ZKClient from './upstream/zookeeper';
```
Comment (kaidamasaki) [inline lib/boot.js]: What zookeeper client are you using?  Have you talked to Frontend about their problems with Odysseus?

File: lib/router.js [hunk 2]
```diff
@@ -4,38 +4,25 @@
...
 import Spatial from './services/spatial';
 import Summary from './services/summary';
+import logger from './util/logger';
 
 function js(req, res, next) {
-  req.setEncoding('binary');
-  req.on('data', (chunk) => {
-    // console.log("chunk", typeof chunk) //TODO: log this
-  });
+  // req.setEncoding('binary');
```
Comment (kaidamasaki) [inline lib/router.js]: Remove commented code, or replace it with a comment if this is intended as documentation.

File: lib/services/spatial.js [hunk 3]
```diff
@@ -98,7 +98,7 @@ class SpatialService {
         };
 
         //logthis
```
Comment (kaidamasaki) [inline lib/services/spatial.js]: Consider removing?

File: lib/upstream/core.js [hunk 4]
```diff
@@ -19,7 +20,7 @@ class CoreAuth {
   }
 
   get cookie() {
-    return this._headers['cookie'];
+    return this._headers.cookie;
```
Comment (kaidamasaki) [inline lib/upstream/core.js]: :cookie:

**Comment by kaidamasaki** [comment]
This should have been multiple commits.

Otherwise seems pretty reasonable generally.

File: lib/boot.js (hunk 1 above)
Comment (rozap) [inline lib/boot.js]: using `node-zookeeper-client`

**Comment by rozap** [comment]
fixed in the next commit :+1:

**Pull request merged**